packages/@aws-cdk/aws-s3/test/integ.bucket-auto-delete-objects.ts

```diff
@@ -0,0 +1,18 @@
+import { App, RemovalPolicy, Stack, StackProps } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as s3 from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    new s3.Bucket(this, 'Bucket', {
+      removalPolicy: RemovalPolicy.DESTROY,
+      autoDeleteObjects: true,
+    });
+  }
```

Comment:
How about we add a custom resource that writes something to the bucket? So that the integ test will fail if the auto-delete handler is malfunctioning.

Reply:
Good idea